rke/k8s_rke_system_images.go

```diff
@@ -2737,6 +2772,41 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.3"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.7"),
 		},
+		// Enabled in Rancher v2.3.8
```

Comment:
should this say enabled out of band?

Reply:
Yes, we should indicate it was enabled out of band after v2.3.8